tournaments/index.js

```diff
@@ -144,6 +159,41 @@ class Tournament {
 		return true;
 	}
 
+	getBanlist() {
+		let bans = [];
+		let unbans = [];
+		let addedRules = [];
+		let removedRules = [];
+		for (let i = 0; i < this.banlist.length; i++) {
+			let ban = this.banlist[i];
+			let unban;
```

Comment:
This makes me mildly nervous because this would be a bug if it was `var` instead of `let`.

I don't think we should ever rely on an uninitialized variable being `undefined`, especially since I think `= false` makes the intent of this code much clearer.